lib/plugin.js

```diff
@@ -1,37 +1,34 @@
-module.exports = ({ app, query, $axios, $graphql }) => {
-  // axios interceptor
+module.exports = ({ app, query, $axios, $graphql, enablePreview }) => {
+  // Add Axios interceptor
   if ($axios) {
     $axios.onRequest(config => {
-      if (query.preview) {
+      if (query.preview && query.token) {
         if (config.params) {
-          config.params.preview = query.preview
+          config.params.token = query.token
         } else {
           config.params = {
-            preview: query.preview
+            token: query.token
           }
         }
       }
+
       return config
     })
   }
 
-  // set graphql
+  // Update GraphQL endpoints
   if ($graphql) {
     app.router.afterEach((to, from) => {
-      if (to.query.preview) {
+      if (to.query.preview && to.query.token) {
         Object.keys($graphql).forEach(key => {
-          $graphql[key].url = `${$graphql[key].url}?preview=${to.query.preview}`
+          $graphql[key].url = `${$graphql[key].url}?token=${to.query.token}`
         })
       }
     })
   }
 
-  // live update event handler
-  if (process.client) {
-    window.onmessage = function (e) {
-      if (e.data === 'liveUpdate') {
-        window.$nuxt.refresh()
-      }
-    }
+  // Enable preview mode for static sites
+  if (process.client && query.preview) {
+    enablePreview()
```

Comment:
Ist diese Funktion immer verfügbar? In der [Doku](https://nuxtjs.org/docs/features/live-preview/#:~:text=Only%20available%20when%20using%20target%3Astatic) steht:
> Only available when using `target:static`
Falls sie einfach nichts macht, kann man es schon so lassen 👍🏻

Reply:
fixed